package.json

```diff
@@ -45,7 +45,7 @@
     "@nomicfoundation/hardhat-network-helpers": "^1.0.8",
     "@nomicfoundation/hardhat-toolbox": "2.0.2",
     "@nomiclabs/hardhat-ethers": "2.2.2",
-    "@nomiclabs/hardhat-etherscan": "3.0.0",
+    "@nomiclabs/hardhat-etherscan": "3.1.7",
```

Comment:
you don't get an error with this? I thought this package is already part of the `hardhat-toolbox`, no?

Reply:
I had an error without adding this, and this was the resolution. I couldn't setup a Sepolia network anymore in the HH config without upgradeing the package